### DISCUSSION TITLE
Is there anyone who uses fo-Dicom Asset in Unity?

### DISCUSSION THREAD

**Discussion by JHsGame** [fo-dicom/fo-dicom#1394] [Q&A]
Hello, it's a good day. I would appreciate it if you could understand that I use a translator because I can't speak English.

The reason why I'm writing this is because there's a problem with using the fo-Dicom set.

The version of fo-Dicom Unity you are using is a little far from the latest version of fo-Dicom.

So there is a problem that some DICOM files cannot be loaded.

If you know how to solve this problem or upgrade the fo-Dicom Unity Asset to the latest fo-Dicom, I would appreciate it if someone could help me.

**Comment by mrbean-bremen** [comment]
Duplicate of #1346 (unfortunately still not answered).
